### PR TITLE
[T2A2][W13-A4]Lai Hoang Dung

### PR DESCRIPTION
**Pull request by louislai**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -210,12 +210,7 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
-        }
+        processUserInput();
```
Comment (K1ang) [inline src/seedu/addressbook/AddressBook.java:213]: Good attempt in refactoring the main method to improve SLAP.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -226,6 +221,15 @@ public static void main(String[] args) {
      * ====================================================================
      */
 
+    private static void processUserInput() {
```
Comment (K1ang) [inline src/seedu/addressbook/AddressBook.java:224]: Missing header comment. All non trivial methods should have javadoc format header comments.

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -1155,6 +1159,13 @@ private static String removePrefixSign(String s, String sign) {
     }
 
     /**
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
+     */
```
Comment (K1ang) [inline src/seedu/addressbook/AddressBook.java:1163]: Normally, you should give a description of the `@param` or the `@return` value.

**Review by K1ang — COMMENTED** [review]
@louislai Some comments added. Please close the PR after reading comments.

**Comment by louislai** [comment]
@K1ang comments acknowledged. PR closed!

**Pull request closed without merging**